lib/semian/atomic_integer.rb

```diff
@@ -0,0 +1,28 @@
+module Semian
+  class AtomicInteger #:nodoc:
```

Comment:
Once we're on the SysV implementation we'll want to create a Thread Safe variant of this as well.